Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj

```diff
@@ -337,7 +340,7 @@
 				TargetAttributes = {
 					13A342EA1F0367E9002EBC92 = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 5BL5M2XBM2;
+						DevelopmentTeam = LA4QFXA5C5;
```

Comment:
Okay, I'm not sure the best way for all that to happen is, iOS code signing is confusing. But thats fine, it is still working for me, and it runs on your device. I suspect we will end up revisiting this but for now this is good enough for me.